bowtie/_cli.py

```diff
@@ -761,6 +761,14 @@ def _check_fail_fast_provided(
     return value
 
 
+def _validate_schema_and_dialect(
+    schema: Any,
+    dialect: Any,
+) -> bool:
+    # here we will verify the condition and return true or false based on that
```

Comment:
@Julian is there a way were we can access the pretty name from $schema property (which is a URL in this case) as the dialect we recieve from the argument is in pretty name format ie `Draft 7`

Reply:
You should spend a bit longer trying to answer this question yourself before asking, as finding the answer I don't suspect should be too challenging, you'll find places in the existing codebase doing this, and also the first place you look should be the `Dialect` object. But yes, going from a URI to a dialect object is `Dialect.by_uri()[uri]` at the minute.

Reply:
My appologies for the inconviniece. But I already wasted 2-3 days in search for the -D in the `suite` subcommand. Wanted to finish this PR and move ahead :sweat_smile: . Sorry the hurried doubts definitely wont reapeat it.